geode-core/src/main/java/org/apache/geode/internal/cache/TombstoneService.java

```diff
@@ -373,7 +373,8 @@ public Object getBlockGCLock() {
     return this.replicatedTombstoneSweeper.getBlockGCLock();
   }
 
-  protected static class Tombstone extends CompactVersionHolder {
+  @VisibleForTesting
+  public static class Tombstone extends CompactVersionHolder {
```

Comment:
To prevent having to make this class and its fields/methods public, would it be possible to move TombstoneDUnitTest to the same package as this one? Or to move this class to the same package as TeombstoneDUnitTest?

Reply:
They are only public for test. Other than that, I am not sure let me look at it.